e2e/portalicious/pages/RegistrationsPage.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, Locator } from '@playwright/test';
+import { Page } from 'playwright';
+
+import BasePage from './BasePage';
+
+class RegistrationsPage extends BasePage {
+  readonly page: Page;
+  readonly tableRows: Locator;
+  readonly selectAllPasCheckbox: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.tableRows = this.page.locator('table tbody tr');
+    this.selectAllPasCheckbox = this.page.getByRole('cell', {
+      name: 'All items unselected',
+    });
+  }
+
+  async selectAllPAs() {
+    await this.selectAllPasCheckbox.click();
+  }
+
+  async selectBulkAction(action: string) {
+    await this.page.getByRole('button', { name: action }).click();
+  }
+
+  async selectCustomMessage() {
+    await this.page
+      .locator('label')
+      .filter({ hasText: 'Custom message' })
+      .click();
+  }
+
+  async typeMessage(message: string) {
```

Comment:
```suggestion
  async typeCustomMessage(message: string) {
```

Just slightly clearer I suppose

Reply:
Done!